hydrabus_framework/core/engine.py

```diff
@@ -0,0 +1,122 @@
+import inspect
+import os
+import pkgutil
+import sys
+from hydrabus_framework.core.logger import Logger
```

Comment:
Your own package should be the latest included to prevent overwritting issue

Reply:
Done